components/TeamCard.tsx

```diff
@@ -81,12 +81,44 @@ export function TeamCard(team: Team & { userRank?: number, url: string, register
         }
     }
 
+    const kickMember = (id: string) => {
```

Comment:
I would take in the entire member as just from this it's difficult to see if the id is team specific or the member id.